memcached-operator/README.md

```diff
@@ -53,50 +43,30 @@ $ operator-sdk build $IMAGE
 $ docker push $IMAGE
 ```
 
-### Run the tests
-
-To run the tests in `test/e2e`, first make sure to update the image name from `REPLACE_IMAGE`
-to your desired image name in `deploy/operator.yaml`:
+### Using the image
 
 ```
-$ sed "s@REPLACE_IMAGE@$IMAGE@g" -i deploy/operator.yaml
+# Update the operator manifest to use the built image name (if you are performing these steps on OSX, see note below)
+$ sed -i 's|REPLACE_IMAGE|quay.io/example-inc/memcached-operator|g' deploy/operator.yaml
+# On OSX use:
+$ sed -i "" 's|REPLACE_IMAGE|quay.io/example-inc/memcached-operator|g' deploy/operator.yaml
 ```
 
-Run the tests that reside in `test/e2e`:
+**NOTE** The `quay.io/example-inc/memcached-operator` is an example. You should build and push the image for your repository. 
 
-```
-$ operator-sdk test local ./test/e2e
-```
+### Installing
 
-To run go-test with verbose and limit to 2 parallel tests:
+Run `make install` and check the operator and the example Memcached deployed and running in the cluster 
```

Comment:
Run `make install` to install the operator. Check that the operator is running in the cluster, also check that the example Memcached service was deployed.

Reply:
Terrific suggestion. Done 👍 tks